lib/src/constants/color_constants.dart

```diff
@@ -8,4 +8,9 @@ class ColorConstants {
     end: Alignment.topLeft,
     colors: [Color(0xFF17EAD9), Color(0xFF6078EA)],
   );
+  static const Color colorTitleDetailScreen = Colors.white;
+  static const Color colorIconCoinName = Colors.orange;
+  static const Color colorCoinPricePlus = Colors.green;
+  static const Color colorHigh = Colors.green;
+  static const Color colorLow = Colors.red;
```

Comment:
do we need constants for these when simple translations with `copyWith()` in screen is enough?

Reply:
I think it will be useful in a long code when I want to find and edit color. Now, I understand, this project doesn't need it. Thank you !